scripts/irods/test/test_specific_queries.py

```diff
@@ -0,0 +1,24 @@
+import os
+import sys
+import unittest
+
+from . import session
+
+class Test_Specific_Queries(session.make_sessions_mixin([('otherrods', 'rods')], []), unittest.TestCase):
+
+    def setUp(self):
+        super(Test_Specific_Queries, self).setUp()
+        self.admin = self.admin_sessions[0]
+
+    def tearDown(self):
+        super(Test_Specific_Queries, self).tearDown()
+
+    def test_DataObjInCollReCur__issue_6900(self):
+        collection = os.path.join(self.admin.session_collection, 'test.d')
+        self.admin.assert_icommand(['imkdir', 'test.d'])
```

Comment:
Would it be worth running this as a `rodsuser` in addition to/instead of `rodsadmin`? We've run into some issues in the past by only testing for `rodsadmin`s. Totally fine if not

Reply:
I don't think it is necessary, but I do think that will improve the test and provide some future proofing for us.

Will change it to a rodsuser.

Reply:
Done.